server/src/main/java/io/spine/server/bus/Bus.java

```diff
@@ -74,7 +76,17 @@ public void unregister(D dispatcher) {
      * @param message the message to post
      * @param responseObserver the observer to receive outcome of the operation
      */
-    public abstract void post(T message, StreamObserver<Response> responseObserver);
+    public void post(T message, StreamObserver<Response> responseObserver) {
```

Comment:
I also don't get where the overloaded signature is. We should have a common contract for all of `Bus`es to enable grouped `Command` dispatching in future (not within this PR).